testscpp/tests/test_roof_duality.cpp

```diff
@@ -52,6 +52,19 @@ TEST_CASE("Tests for fixQuboVariables", "[roofduality]") {
         }
     }
 
+    SECTION("Test offset") {
+        float Q [9] = {22,-4, 0,
+                        0, 0, 4,
+                        0, 0,-2};
+
+        int num_vars = 3;
+        auto bqm = dimod::AdjVectorBQM<int, float>(Q, num_vars);
+
+        auto result = fixQuboVariables(bqm, true, -2);
```

Comment:
auto bqm_offset = -2;
        auto result = fixQuboVariables(bqm, true, bqm_offset);
        auto lower_bound = result.first;
        REQUIRE(lower_bound == (-2 + bqm_offset));
        
        Makes it more elaborate, I guess.